test_windows.js

```diff
@@ -0,0 +1,31 @@
+var Mocha = require('mocha')
+var fs = require('fs');
+var path = require('path');
+var semver = require('semver');
+
+
+var mocha = new Mocha();
+var testDir = 'build/test';
+fs.readdirSync(testDir).filter(function(file) {
+    return file.substr(-3) === '.js';
+}).forEach(function(file) {
+    mocha.addFile(
+        path.join(testDir, file)
+    );
+});
+
+process.env.GCLOUD_PROJECT = 0
+process.env.CLOUD_DEBUG_ASSERTIONS = 1
+
```

Comment:
Don't you want to call `mocha` here without `GCLOUD_USE_INSPECTOR` set to run the tests without using inspector?

Reply:
I find it hard to run both in one js file. So I leave that in appveyor and have this js just for inspector.